src/components/custom/_banner-alert.tsx

```diff
@@ -0,0 +1,231 @@
+        cookie_browser_update.set(cookie_key, true, {
+            expires: new Date(expiration_date),
+        })
+        setIsVisible(false)
+    }
+
+    const generateBanner = useMemo(() => {
+        switch (bannerType) {
+            case bannerTypes.cookieBanner:
+                return (
+                    <Wrapper
+                        visible={cookie.should_show}
+                        width={cookieBannerProps.width}
+                        minHeight={cookieBannerProps.minHeight}
+                    >
+                        <StyledText>
+                            <Localize translate_text="_t_Cookies help us to give you a better experience and personalised content on our site._t_" />
+                            <Localize
+                                translate_text="_t_If you agree to our use of cookies, click on Accept. For more information, <0>see our policy</0>._t_"
+                                components={[
+                                    <LinkText
+                                        key={0}
+                                        to="/terms-and-conditions/#clients"
+                                        color="red"
+                                    />,
+                                ]}
+                            />
+                        </StyledText>
+                        <Flex>
+                            <StyledButton tertiary onClick={cookie.decline} mr="0.8rem">
+                                {localize("_t_Don't accept_t_")}
+                            </StyledButton>
+                            <StyledButton secondary onClick={cookie.accept}>
+                                {localize('_t_Accept_t_')}
+                            </StyledButton>
+                        </Flex>
+                    </Wrapper>
+                )
+
+            case bannerTypes.outdatedBrowserBanner:
+                return (
+                    <OverlayContainer is_rtl={is_rtl}>
+                        <Wrapper
+                            visible={is_visible}
+                            width={outdatedBrowserBannerProps.width}
+                            minHeight={outdatedBrowserBannerProps.minHeight}
+                        >
+                            <StyledText>
+                                <Localize translate_text="_t_Update your browser to get the best Deriv experience_t_" />
+                            </StyledText>
+                            <Flex>
+                                <StyledButton secondary onClick={handlePositive}>
+                                    <Localize translate_text="_t_Close_t_" />
+                                </StyledButton>
+                            </Flex>
+                        </Wrapper>
+                        {/* TODO:not sure about this bug fix have to confirm  (bug:when CFD banner is alraedy there this banner should show in top of that) */}
+                        {(is_eu || is_cpa_plan) && is_tablet ? (
```

Comment:
when CFD banner is their browser banner is not visible so i tried a solution not sure have to check this